velox/functions/prestosql/aggregates/tests/PrestoHasherTest.cpp

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/prestosql/aggregates/PrestoHasher.h"
+#include "velox/functions/prestosql/aggregates/tests/AggregationTestBase.h"
+
+namespace facebook::velox::aggregate::test {
+using namespace facebook::velox::aggregate;
+
+class PrestoHasherTest : public AggregationTestBase {
+ protected:
+  template <typename T>
+  void assertHash(
```

Comment:
It would be easier to read if this method appears next to the other assertHash() method and if the order and names of the parameters were consistent, e.g. data, expected.